crates/ruff_linter/src/rules/pyflakes/rules/unused_import.rs

```diff
@@ -46,16 +46,32 @@ enum UnusedImportContext {
 /// from module import member as member
```

Comment:
Just realized I was commenting on a test snapshot :) changes refer to this file.

Reply:
No worries :) Thanks for your suggestions.